src/Toolkit/A.php

```diff
@@ -588,7 +588,7 @@ public static function pluck(array $array, string $key): array
 	 */
 	public static function prepend(array $array, array $prepend): array
 	{
-		return $prepend + $array;
+		return static::merge($prepend, $array, A::MERGE_APPEND);
```

Comment:
Thinking about it, @bastianallgeier - maybe even better to reuse the append one, writing: 

```php
return state::append($prepend, $array);
```